tests/test_imageDecorrelation.py

```diff
@@ -122,17 +122,18 @@ def makeFakeImages(size=(256, 256), svar=0.04, tvar=0.04, psf1=3.3, psf2=2.2, of
     im1 = np.random.normal(scale=np.sqrt(svar), size=x0im.shape)  # variance of science image
```

Comment:
It's slightly out of scope, but since you're modifying `makeFakeImages` it would be helpful if you could convert the docstring to numpydoc format (commenting here because I can't leave a comment on line 77).